src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import circleShadow from './CIRCLE_BUTTON_SHADOW.svg';
+import linkedinIcon from './LINKEDINLOGO.svg';
+import githubIcon from './GITHUBLOGO.svg';
+
+class Footer extends Component {
+  render() {
+    return (
+      <footer className="wrapper pt-5 pb-5 flex justify-between items-center">
+        <div className="flex">
+          <h4>Open Library Project</h4>
+        </div>
+        <div className="links flex">
+          <ShowOffLink
+            link="https://www.linkedin.com/in/julia-lepner/"
+            icon={circleShadow}
+            logo={linkedinIcon}
+          />
+          <ShowOffLink
+            link="https://github.com/ylepner/"
+            icon={circleShadow}
+            logo={githubIcon}
+          />
+        </div>
+        <a href="https://github.com/ylepner" target="_blank" rel="noreferrer">
+          <h4>ylepner</h4>
+        </a>
+        <h4>2023</h4>
+      </footer>
+    );
+  }
+}
+export default Footer;
+
+class ShowOffLink extends Component<{
+  link: string;
+  icon: string;
+  logo: string;
+}> {
+  render(): React.ReactNode {
+    return (
+      <div className="relative pr-5">
+        <a href={this.props.link} target="_blank" rel="noreferrer">
+          <img src={this.props.icon} alt="circle shadow" />
```

Comment:
Those icons are decorative. So it is better to make alt empty, just `alt=""`. And for the better accessibility it is better to add `aria-label` to the link with the text describing the link e.g. `aria-label="Go to ylepner instagram page"`

Reply:
Ok, changed it